web/src/components/nav/Navbar.tsx

```diff
@@ -70,6 +83,27 @@ export const Navbar: React.FC = () => {
               >
                 <CommonFateLogo h="32px" w="auto" />
               </ChakraLink>
+              <Button
+                size="md"
+                variant="unstyled"
+                bg="neutrals.100"
+                rounded="full"
+                border="1px solid"
+                borderColor="neutrals.200"
+                aria-label="Search"
+                // w="113px"
+                // px={2}
+                textAlign="left"
+                color="neutrals.600"
+                onClick={modal.onOpen}
+              >
+                <SearchIcon color="neutrals.600" boxSize="15px" ml={3} mr={2} />
+                Search
+                <Flex ml={2} mr={3} display="inline-flex">
+                  <Kbd>⌘</Kbd>
```

Comment:
We need to support users of windows and linux, at the moment this only supports mac users who have the command key.

The chakra docs use something like this, however navigator.platform is deprecated, so I would recommend using a different approach, there are a few examples online for this.
https://github.com/chakra-ui/chakra-ui-docs/blob/main/src/components/algolia-search.tsx
```typescript
const ACTION_KEY_DEFAULT = ['Ctrl', 'Control']
const ACTION_KEY_APPLE = ['⌘', 'Command']

const [actionKey, setActionKey] = React.useState<string[]>(ACTION_KEY_APPLE)
  React.useEffect(() => {
    if (typeof navigator === 'undefined') return
    const isMac = /(Mac|iPhone|iPod|iPad)/i.test(navigator.platform)
    if (!isMac) {
      setActionKey(ACTION_KEY_DEFAULT)
    }
  }, [])
```



```suggestion
                  <Kbd>⌘</Kbd>
```